src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -36,12 +36,26 @@ class ReportActionItemMessageEdit extends React.Component {
         this.debouncedSaveDraft = _.debounce(this.debouncedSaveDraft.bind(this), 1000, true);
         this.publishDraft = this.publishDraft.bind(this);
         this.triggerSaveOrCancel = this.triggerSaveOrCancel.bind(this);
+        this.onSelectionChange = this.onSelectionChange.bind(this);
 
         this.state = {
             draft: this.props.draftMessage,
+            selection: {
+                start: this.props.draftMessage.length,
+                end: this.props.draftMessage.length,
+            },
         };
     }
 
+    /**
+     * Update Selection on change cursor position.
+     *
+     * @param {Event} e
+     */
+    onSelectionChange(e) {
```

Comment:
Overall looks good but this state change is unnecessary, I have already seen a drawback of this and I am migrating tis unnecessary state change. So my suggestion here would be to remove this   as I mentioned over the issue.
cc: @Gonals

Reply:
@parasharrajat can you please let me know about the issue?
also, without state change, this issue will not be fixed.

Reply:
This change is going against our old pattern of not making the Component controlled.